core/trino-main/src/test/java/io/trino/operator/output/TestPositionsAppender.java

```diff
@@ -241,14 +253,34 @@ public void testSliceRle()
         }
     }
 
+    @Test
+    public void testSliceAppenderInputNotByteArray()
+    {
+        testAppend(TestType.VARCHAR, ImmutableList.of(
+                input(new VariableWidthBlock(3, Slices.wrappedLongArray(257, 2), new int[] {0, 1, Long.BYTES, 2 * Long.BYTES}, Optional.empty()), 0, 2)));
+    }
+
+    @Test
```

Comment:
rename, `NotByteArray` -> `WithSomeType`

Reply:
I dropped this test. it's no longer needed.